code/modules/spell_system/spells/spell_list/targeted_spells.dm

```diff
@@ -142,4 +144,11 @@ Targeted spells have two useful flags: INCLUDEUSER and SELECTABLE. These are exp
 	target.eye_blurry += amt_eye_blurry
 	target.dizziness += amt_dizziness
 	target.confused += amt_confused
-	target.stuttering += amt_stuttering
\ No newline at end of file
+	target.stuttering += amt_stuttering
+	if(ishuman(target))
+		var/mob/living/carbon/human/H = target
+		for(var/obj/item/organ/O in H.internal_organs)
+			if(amt_organ > 0)
+				O.take_damage(amt_organ)
+			else
+				O.heal_damage(amt_organ)
```

Comment:
Pretty sure that doesnt work as amt_organ is negative and these procs expect a positive value